engine/src/main/java/org/terasology/engine/subsystem/headless/assets/HeadlessTexture.java

```diff
@@ -85,7 +84,7 @@ public Texture getTexture() {
 
     @Override
     public Rectanglef getRegion() {
```

Comment:
I think this should ideally return `Rectanglefc`, but this is rooted somewhere in NUI...